src/python/py/models/builder.py

```diff
@@ -2036,13 +2048,14 @@ def make_model(self, input_path):
                 kv_size,
                 self.intermediate_size,
                 self.num_layers,
+                self.extra_options["adapter_path"] if "adapter_path" in self.extra_options else None,
```

Comment:
Can we put all of the parameters passed to `QuantModel.from_pretrained` back in one line?